### ISSUE TITLE
Jihad Sound Bug

### ISSUE THREAD

**Issue by Maxi605** [GuskiS/Trouble-in-Terrorist-Town#6]
If you active the Jihad under the water the sound of the jihad is not reproduced for anyone, this is a popular "exploit" for the item.

**Comment by GuskiS** [comment]
Will check 👍

**Comment by GuskiS** [comment]
I wonder, can you hear the sound yourself and noone else can hear it? Or noone can hear it? Currently trying to reproduce this one.

**Comment by GuskiS** [comment]
Hmm. When I activate the Jihad, I can hear the sound myself.

**Comment by Maxi605** [comment]
Yes, but if you do it under the water the other players can't hear the jihad

**Comment by GuskiS** [comment]
Ok, this will require some testing help cause I can't do this alone.

**Comment by Maxi605** [comment]
Sure, if you have a server i can help you, otherwise i have a TTT Test server with modified and base plugins from the mod itself, the only thing it's that you're goin to have a bit of lag haha

**Comment by GuskiS** [comment]
I might be able to setup a simple server, but not today. Add me on steam and I will let you know when this is needed. I think my steam name is `guskis`.

Btw, please check pull requests I'm creating, see if the code seems correct (I haven't done pawn in a while), also if you could test the code against your server, that would be great. Write in those PRs if you have tested/have something in mind.

**Comment by Maxi605** [comment]
I added you on Steam, i need to ask you something
